src/components/common/listOfArticles/index.js

```diff
@@ -8,8 +8,6 @@ export default function ListOfArticles({ articles, limitArticlesNumber }) {
     return (
       <a
         key={article && article.slug}
-        href={`/article/${article
-          && article.slug}`}
       >
```

Comment:
Why was the `href` removed it?

Reply:
Becouse it was setting whole article card a link to article,including author tho I was supposed to get single article when I click it's header image or article title

Reply:
Do you think that It will be able to be clickable and take a user to view single article if the whole card is not a link? If yes that would be great

Reply:
yeah by clicking article title or article image @nshutijonathan

Reply:
`a` tags require `href` if you remove it there will be errors. You can change it to `div`

Reply:
I have changed it to div thanks